test/e2e_node/pod_hostnamefqdn_test.go

```diff
@@ -140,4 +148,53 @@ var _ = SIGDescribe("Hostname of Pod [Feature:SetHostnameAsFQDN][NodeAlphaFeatur
 		f.TestContainerOutput("fqdn and fqdn", pod, 0, output)
 	})
 
+	/*
+	   Release: v1.19
+	   Testname: Fail to Create Pod with longer than 64 bytes FQDN when setHostnameAsFQDN field set to true.
+	   Description: A Pod that defines the subdomain field in it spec has FQDN.
+	                 When setHostnameAsFQDN: true, the hostname is set to be
+	                 the FQDN. Since kernel limit is 64 bytes for hostname field,
+	                 if pod FQDN is longer than 64 bytes it will generate events
+	                 regarding FailedCreatePodSandBox.
+	*/
+
+	ginkgo.It("a pod configured to set FQDN as hostname will remain in Pending "+
```

Comment:
nit: this test is not checking that Pod actually in pending state. It only checks for event.

Reply:
Right, I added a check to ensure pod is in Pending state. I also realized that I forgot to delete the pod, I am now deleting pod, but in my local testing it takes a few seconds to delete this pod, so e2e test is marked as SLOW. Just wanted to make sure it is OK.

Reply:
are other tests delete pods?

Reply:
yes, other tests delete pods, but they go away fast. Maybe it has to do with the fact that this pod will be in backoff trying to create Sandbox?